Fixing pnpm issues

ethanbrooks opened the pull request:
Fixing pnpm issues

naimo84 commented:
Hi Ethan,

may I ask, why you pushed so much releases? 

And please have a look at https://www.conventionalcommits.org/en/v1.0.0/. With standard-version and conventional commit messages, the changelog.md is filled with helpful change messages 😉 

Thanks and best regards,
Benjamin

ethanbrooks commented:
Sorry, about that I was having conflict issues with my package manger's
lock file and your npm build publish.  It will not happen again... Just
banging my head on the build script which requires a version tag and commit
to run.

I added config and cleaned up some naming issues and will resolve anything
hanging in the changelog.md.

Best Regards,
Ethan Brooks





On Mon, Feb 3, 2020 at 12:49 AM Benjamin <notifications@github.com> wrote:

> Hi Ethan,
>
> may I ask, why you pushed so much releases?
>
> And please have a look at https://www.conventionalcommits.org/en/v1.0.0/.
> With standard-version and conventional commit messages, the changelog.md is
> filled with helpful change messages 😉
>
> Thanks and best regards,
> Benjamin
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/naimo84/node-red-contrib-dockerode/pull/9?email_source=notifications&email_token=AD6Y2DPQXWWBAUL6V3FN6ODRA6V65A5CNFSM4KN6RJSKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEKSSQCQ#issuecomment-581249034>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD6Y2DLYYPICWLPMDURBLQDRA6V65ANCNFSM4KN6RJSA>
> .
>


-- 
Ethan S. Brooks

"This message and any attachments are solely for the intended recipient and
may contain confidential or privileged information. If you are not the
intended recipient, any disclosure, copying, use, or distribution of the
information included in this message and any attachments is prohibited. If
you have received this communication in error, please notify me by reply
e-mail and immediately and permanently delete this message and any
attachments."
